test/starlark_tests/rules/apple_verification_test.bzl

```diff
@@ -38,9 +38,16 @@ load(
 
 def _apple_verification_transition_impl(settings, attr):
     """Implementation of the apple_verification_transition transition."""
+
+    # This was added because this transition is also used by
+    # `infoplist_contents_test` and has no "macos_cpus" attribute.
+    macos_cpus = "x86_64"
```

Comment:
hrm prob wrong, but we can fix later

Reply:
Ahh, probably. Let's fix it after if it is.